test_ollama.py

```
@@ -0,0 +1,66 @@
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)